googlemock/test/gmock-internal-utils_test.cc

```diff
@@ -39,6 +39,7 @@
 #include <string>
 #include <sstream>
 #include <vector>
+#include <type_traits>
```

Comment:
Can you keep the alphabetical order please?

Reply:
I tried to fix this, but it leads to changes in the whole order, since the previous code wasn't ordered. If necessary I can order them all.

Reply:
I've just reminded myself I do it in my pull requests :D